substrafl/remote/register/manage_dependencies.py

```diff
@@ -0,0 +1,206 @@
+        dependencies: Dependency object from which the list of local installable dependencies (wheels or modules) will
+            be extracted.
+
+    Returns: list of wheel paths relative to `path`
+    """
+    path.mkdir(exist_ok=True)
+    with tempfile.TemporaryDirectory(dir="/tmp") as tmp_dir_name:
+        tmp_dir = Path(tmp_dir_name)
+        dependencies_paths = dependencies.copy_dependencies_local_package(dest_dir=tmp_dir)
+        wheel_paths = []
+        for dependency in dependencies_paths:
+            if dependency.__str__().endswith(".whl"):
+                wheel_paths.append(dependency)
+                shutil.copy(tmp_dir / dependency, path)
+            else:
+                wheel_name = build_user_dependency_wheel(
+                    Path(dependency),
+                    operation_dir=tmp_dir,
+                )
+                wheel_paths.append(wheel_name)
+                shutil.copy(tmp_dir / wheel_name, path)
+    return wheel_paths
+
+
+def local_lib_wheels(lib_modules: List, *, dest_dir: Path) -> List[str]:
+    """Generates wheels for the private modules from lib_modules list and returns the list of names for each wheel.
+
+     It first creates the wheel for each library. Each of the libraries must be already installed in the correct
+     version locally. Use command: ``pip install -e library-name`` in the directory of each library.
+     Then it copies the wheels to the dest_dir.
+
+    This allows one user to use custom version of the passed modules.
+
+    Args:
+        lib_modules (list): list of modules to be installed.
+        dest_dir (str): relative directory where the wheels are saved
+
+    Returns:
+        List[str]: wheel names for the given modules
+    """
+    wheel_names = []
+    dest_dir.mkdir(exist_ok=True, parents=True)
+    for lib_module in lib_modules:
+        if not (Path(lib_module.__file__).parents[1] / "setup.py").exists():
+            msg = ", ".join([lib.__name__ for lib in lib_modules])
+            raise NotImplementedError(
+                f"You must install {msg} in editable mode.\n" "eg `pip install -e substra` in the substra directory"
+            )
+        lib_name = lib_module.__name__
+        lib_path = Path(lib_module.__file__).parents[1]
+        wheel_name = f"{lib_name}-{lib_module.__version__}-py3-none-any.whl"
+
+        wheel_path = LOCAL_WHEELS_FOLDER / wheel_name
+        # Recreate the wheel only if it does not exist
+        if wheel_path.exists():
+            logger.warning(
+                f"Existing wheel {wheel_path} will be used to build {lib_name}. "
+                "It may lead to errors if you are using an unreleased version of this lib: "
+                "if it's the case, you can delete the wheel and it will be re-generated."
+            )
+        else:
+            # if the right version of substra or substratools is not found, it will search if they are already
+            # installed in 'dist' and take them from there.
```

Comment:
```suggestion
            # installed in 'dist' and take them from there.
```
The lib are not in "dist" anymore if I'm right ?

Reply:
Target tree structure is:
```
├── Dockerfile
├── function.py
├── function.tar.gz
└── substrafl_internal
    ├── cls_cloudpickle
    ├── description.md
    ├── dist
    │   ├── substra-0.44.0-py3-none-any.whl
    │   ├── substrafl-0.36.0-py3-none-any.whl
    │   └── substratools-0.20.0-py3-none-any.whl
    ├── local_dependencies
    │   └── local-module-1.6.1-py3-none-any.whl
    ├── requirements.in
    └── requirements.txt
```

Reply:
I feel like maybe I should document it somewhere, but I'm unsure where. Do you have any suggestion?